.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateParser.cs

```diff
@@ -512,11 +516,11 @@ private DateTimeResolutionResult Match2Date(Match match, DateObject referenceDat
                 if (!string.IsNullOrEmpty(yearStr))
                 {
                     year = int.Parse(yearStr);
-                    if (year < 100 && year >= 90)
+                    if (year < 100 && year >= Constants.MinTwoDigitYearPastNum)
```

Comment:
This method could be somewhere shared between the Chinese parser and the base parser. It would make it simpler to keep implementations consistent.